content/api/ab-testing.apib

```diff
@@ -417,6 +417,41 @@ Returns the `status`.
             ]
         }
 
+### Delete an A/B Test [DELETE /ab-test/{id}]
+
++ Parameters
+    + id (required, string, `password-reset`)
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 200 (application/json)
+
```

Comment:
what are you thinking for 200 response here, just some simple message that says 'ab test was deleted' or something?

Reply:
I was just going to return a 200, I guess I should look at what we normally do for deletes

Reply:
Looks like in other situations we just return `{}`, we can do that

Reply:
we do 204's on deletes